Source/Core/Core/PowerPC/MMU.cpp

```diff
@@ -930,8 +930,10 @@ bool MMU::IsOptimizableRAMAddress(const u32 address) const
   //
   // We store whether an access can be optimized to an unchecked access
   // in dbat_table.
-  u32 bat_result = m_dbat_table[address >> BAT_INDEX_SHIFT];
-  return (bat_result & BAT_PHYSICAL_BIT) != 0;
+  const u32 last_byte_address = address + (access_size >> 3) - 1;
```

Comment:
There's a TODO a few lines up that you can remove now.

Reply:
Also is this even correct? Why the bitshift by 3?

Reply:
Oh is the size given in bits, not bytes? In that case yeah that makes sense then.

Reply:
> There's a TODO a few lines up that you can remove now.

Oops, yes.

> Oh is the size given in bits, not bytes? In that case yeah that makes sense then.

Yeah, it's in bits, following the pattern used by IsOptimizableMMIOAccess.